Assets/LeapMotion/Core/Scripts/Utils/RuntimeGizmoManager.cs

```diff
@@ -656,38 +729,115 @@ public class RuntimeGizmoDrawer {
       PopMatrix();
     }
 
+    public void DrawCollider(Collider collider, bool useWireframe = true) {
```

Comment:
DrawColliders should probably be refactored to use this method

Reply:
I didn't do that because it appeared to be strictly less efficient to do it that way, but happy to make the change anyway in the interest of avoiding code duplication

Reply:
Hmm if there is indeed a performance difference, I'd want to see if we could remove the duplication by another means (maybe by having a dedicated private method for each collider, so at least _that_ code isn't being duplicated?)

But I'd guess that the performance difference isn't going to be significant, especially compared to the overhead of actually drawing the gizmos in the first place.